src/time/Storage.cpp

```diff
@@ -16,6 +16,7 @@ const double Storage::WINDOW_END = 1.0;
 Storage::Storage()
     : _stampleStorage{}, _degree(0)
 {
+  math::Bspline _bspline = math::Bspline();
 }
```

Comment:
```suggestion
Storage::Storage()
    : _stampleStorage{}, _degree(0), _bspline()
{
}
```

@fsimonis Please correct me if this is nonsense :smile: